src/find_bottleneck.rs

```diff
@@ -199,23 +232,31 @@ impl FindBottleneck {
         }
         Ok(())
     }
-    /// Writes the asn bottleneck result to a stdout or a time stamped file
-    pub(crate) fn write(self, out: Option<&Path>) -> Result<()> {
-        if let Some(path) = out {
-            let epoch = SystemTime::now()
-                .duration_since(SystemTime::UNIX_EPOCH)
-                .unwrap();
-            let now = epoch.as_secs();
-            let dst = path.join(format!("bottleneck.{}.txt", now));
-            let mut file = File::create(&dst).map_err(|io_error| Error::IoError {
-                io_error,
-                path: dst.to_path_buf(),
-            })?;
 
-            self.write_bottleneck(&mut file)?;
+    /// Writes the asn bottleneck result to a time stamped file in a specified or default location
+    pub(crate) fn store_result(temp_result_file: &File, out: Option<&Path>) -> Result<()> {
```

Comment:
Can we leave the name of this function as `write`?

Reply:
yeah i don't care. will update